client/package.json

```diff
@@ -7,9 +7,11 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.3",
     "@testing-library/user-event": "^12.6.2",
+    "node-fetch": "^2.6.1",
```

Comment:
fetch is build-in in browsers this day so no need to install  it,
and  I prefer to install Axios.